js/NewsChannel.js

```diff
@@ -0,0 +1,128 @@
+var bodyDiv;
+var rightPannel, leftPannel;
+const newsChannels =["All","Channel 1","Channel 2","Channel 3","Channel 4","Channel 5",
+"Channel 6", "Channel 7", "Channel 8", "Channel 9","Channel 10"];
+function GetChannelsInfo(){
+  CreateHeader();
+  bodyDiv = document.createElement('div');
+  bodyDiv.setAttribute('class','body');
+  leftPannel = document.createElement('div');
+  leftPannel.setAttribute('class','left-pane');
+  bodyDiv.appendChild(leftPannel);
+  rightPannel = document.createElement('div');
+  rightPannel.setAttribute('class','right-pane');
+  bodyDiv.appendChild(rightPannel);
+  document.body.appendChild(bodyDiv);
+  for(var index =1; index< newsChannels.length; index++)
+  {
+    LoadChannelInfo(index);
+  }
+    CreateCategoryinfo();
+  CreateFooter();
+}
+// Header Information
+function CreateHeader(){
+  var  headerInfo = document.createElement('div');
+  headerInfo.setAttribute('class','header');
+  let headerTag=document.createElement('h2');
+  headerTag.innerHTML="NEWSFEED";
+  headerTag.setAttribute('class','h2');
+  headerInfo.appendChild(headerTag);
+  var para= document.createElement("p");
+  let node=document.createTextNode("Yet Another News Feed");
+  para.setAttribute('class','description');
+  para.appendChild(node);
+  headerInfo.appendChild(para) ;
+  document.body.appendChild(headerInfo);
+  return headerInfo;
+}
+
+//Category & Email Subscribe Information
+function CreateCategoryinfo(){
+  let categoryDiv = document.createElement('div');
+  let  categoryElement = document.createElement('div');
+  let catergoryText = document.createElement('text');
+  catergoryText.innerHTML="SELECT CATEGORY";
+  catergoryText.setAttribute('class','news-category');
+  categoryElement.appendChild(catergoryText);
+  categoryElement.setAttribute('class','category-dropdown');
+  categoryElement.appendChild(catergoryText);
+  categoryDiv.appendChild(categoryElement);
+
+  let selectElement =document.createElement("select");
+  selectElement.setAttribute('class','dropdown');
+  selectElement.setAttribute('OnChange','ShowHideDiv(this)');
+  categoryDiv.appendChild(selectElement);
+
+  for(var index =0 ;index< newsChannels.length; index++){
+    var option= document.createElement("option");
+    option.value= index;
+    option.text= newsChannels[index];
+    selectElement.appendChild(option);
+  };
+  let subscribeDiv =document.createElement('div');
+  let subscribeText = document.createElement('text');
+  subscribeText.innerHTML="SUBSCIBE";
+  subscribeText.setAttribute('class','subscribe');
+  subscribeDiv.appendChild(subscribeText);
+
+  let emailElement = document.createElement('input');
```

Comment:
Validate input email and show the error message below the email text box.

Reply:
Updated the code to show error message as"Please enter valid Email Address"